module/assets/helper.js

```diff
@@ -0,0 +1,786 @@
+            programMatchQuery.owner = { $in: filterEmptyStringsFromArray };
+          }
+        }
+        switch (queryData) {
+          case "program":
+            organizationAssets = await programsHelper.list(
+              "",                // for pageNo
+              "",                // for pageSize
+              "",                // for searchText
+              programMatchQuery,
+              bodyData.fields
+            );
+            break;
+          case "solution":
+            organizationAssets = await solutionsHelper.list(
+              "",                //for type
+              "",                // for subType
+              solutionMatchQuery,
+              "",                // for pageNo
+              "",                // for pageSize
+              "",                // for searchText
+              bodyData.fields
+            );
+            break;
+          default:
+            let allOrganizationProgram = await programsHelper.list(
+              "",               // for pageNo
+              "",               // for pageSize
+              "",               // for searchText
+              programMatchQuery,
+              bodyData.fields
+            );
+            let allOrganizationSolutions =
+              await solutionsHelper.list(
+                "",             //for type
+                "",             // for subType
+                solutionMatchQuery,
+                "",             // for pageNo
+                "",             // for pageSize
+                "",             // for searchText
+                bodyData.fields
+              );
+            organizationAssets = {
+              success: true,
+              message: constants.apiResponses.ASSETS_FETCHED_SUCCESSFULLY,
+              data: {
+                data: [
+                  ...allOrganizationProgram.data.data,
+                  ...allOrganizationSolutions.data.data,
+                ],
+              },
+              count:
+                allOrganizationProgram.data.count +
+                allOrganizationSolutions.data.count,
+            };
+
+            break;
+        }
+
+        return resolve({
+          result: organizationAssets,
+        });
+      } catch (error) {
+        return reject(error);
+      }
+    });
+  }
+
+  /**
+   * Transfer ownership Kafka Event.
+   * @method
+   * @name ownershipTransfer
+   * @param {ownershipTransferEvent} - OwenerShip transfer Event message object
+   * @returns {Promise} success Data.
+   */
+  static ownershipTransfer(ownershipTransferEvent) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let reqData = ownershipTransferEvent.edata;
+        let fromFindQuery = {
+          userId: reqData.fromUserProfile.userId,
+          platformRoles: { $exists: true },
+        };
+
+        let updateUserAssetDataResult = false;
+        let checkUsersRolesIsIdentical = this.checkRolesPresence(
```

Comment:
@praveenKDass Add sample argument value and response expected from the calling function as comments.